test/test_linalg.py

```diff
@@ -3274,8 +3274,21 @@ def test_old_matrix_rank(self, device, dtype):
             self.assertEqual(torch.matrix_rank(aaT, 0.01, True), np.linalg.matrix_rank(aaT.cpu().numpy(), 0.01, True))
 
     @onlyOnCPUAndCUDA
-    @dtypes(torch.float, torch.cfloat)
-    @precisionOverride({torch.float: 1e-02, torch.cfloat: 1e-02})
+    @dtypes(torch.double)
+    def test_chain_matmul(self, device, dtype):
```

Comment:
Add a comment explaining that this test is just checking for chain_matmul()'s additional error checking vs. linalg.multi_dot()